src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -296,9 +296,8 @@ Resizer::ensureWireParasitic(const Pin *drvr_pin,
       parasitics_invalid_.erase(net);
       break;
     case ParasiticsSrc::global_routing: {
-      grt::IncrementalGRoute incr_groute(global_router_, block_);
       global_router_->addDirtyNet(db_network_->staToDb(net));
-      incr_groute.updateRoutes();
+      incr_groute_->updateRoutes();
```

Comment:
Is the call to addDirtyNet still needed?

Reply:
The unit test that checks it didn't fail when I removed it, so I believe it is ok to remove. Added a new secure-ci to ensure it will no break anything.